2022-04-09 12:11:50,914 INFO    MainThread:44436 [wandb_setup.py:_flush():75] Loading settings from /share/etud/e2008984/.config/wandb/settings
2022-04-09 12:11:50,915 INFO    MainThread:44436 [wandb_setup.py:_flush():75] Loading settings from /share/mastoc/projects/erasmus/dl_historical_images/master_thesis/utils/wandb/settings
2022-04-09 12:11:50,915 INFO    MainThread:44436 [wandb_setup.py:_flush():75] Loading settings from environment variables: {}
2022-04-09 12:11:50,915 INFO    MainThread:44436 [wandb_setup.py:_flush():75] Inferring run settings from compute environment: {'program': '<python with no main file>'}
2022-04-09 12:11:50,915 INFO    MainThread:44436 [wandb_init.py:_log_setup():405] Logging user logs to /share/mastoc/projects/erasmus/dl_historical_images/master_thesis/utils/wandb/run-20220409_121150-3q7m5rmy/logs/debug.log
2022-04-09 12:11:50,915 INFO    MainThread:44436 [wandb_init.py:_log_setup():406] Logging internal logs to /share/mastoc/projects/erasmus/dl_historical_images/master_thesis/utils/wandb/run-20220409_121150-3q7m5rmy/logs/debug-internal.log
2022-04-09 12:11:50,915 INFO    MainThread:44436 [wandb_init.py:init():439] calling init triggers
2022-04-09 12:11:50,915 INFO    MainThread:44436 [wandb_init.py:init():442] wandb.init called with sweep_config: {}
config: {}
2022-04-09 12:11:50,916 INFO    MainThread:44436 [wandb_init.py:init():458] re-initializing run, found existing run on stack: 20jbisvc
2022-04-09 12:11:50,918 INFO    MainThread:44436 [wandb_run.py:finish():1539] finishing run vibferreira/master_thesis/20jbisvc
2022-04-09 12:11:50,919 INFO    MainThread:44436 [jupyter.py:save_history():429] not saving jupyter history
2022-04-09 12:11:50,919 INFO    MainThread:44436 [jupyter.py:save_ipynb():374] not saving jupyter notebook
2022-04-09 12:11:50,920 INFO    MainThread:44436 [wandb_init.py:_jupyter_teardown():337] cleaning up jupyter logic
2022-04-09 12:11:50,920 INFO    MainThread:44436 [wandb_run.py:_atexit_cleanup():1702] got exitcode: 0
2022-04-09 12:11:50,928 INFO    MainThread:44436 [wandb_run.py:_restore():1674] restore
2022-04-09 12:11:51,804 INFO    MainThread:44436 [wandb_run.py:_on_finish():1831] got exit ret: file_counts {
  wandb_count: 1
  media_count: 8
}
pusher_stats {
  uploaded_bytes: 39068
  total_bytes: 39068
}

2022-04-09 12:11:52,069 INFO    MainThread:44436 [wandb_run.py:_on_finish():1831] got exit ret: file_counts {
  wandb_count: 1
  media_count: 8
}
pusher_stats {
  uploaded_bytes: 39068
  total_bytes: 39068
}

2022-04-09 12:11:52,833 INFO    MainThread:44436 [wandb_run.py:_on_finish():1831] got exit ret: file_counts {
  wandb_count: 6
  media_count: 8
}
pusher_stats {
  uploaded_bytes: 39068
  total_bytes: 44702
}

2022-04-09 12:11:52,936 INFO    MainThread:44436 [wandb_run.py:_on_finish():1831] got exit ret: file_counts {
  wandb_count: 6
  media_count: 8
}
pusher_stats {
  uploaded_bytes: 39068
  total_bytes: 44702
}

2022-04-09 12:11:53,038 INFO    MainThread:44436 [wandb_run.py:_on_finish():1831] got exit ret: file_counts {
  wandb_count: 6
  media_count: 8
}
pusher_stats {
  uploaded_bytes: 39068
  total_bytes: 44702
}

2022-04-09 12:11:53,140 INFO    MainThread:44436 [wandb_run.py:_on_finish():1831] got exit ret: file_counts {
  wandb_count: 6
  media_count: 8
}
pusher_stats {
  uploaded_bytes: 44702
  total_bytes: 44702
}

2022-04-09 12:11:53,243 INFO    MainThread:44436 [wandb_run.py:_on_finish():1831] got exit ret: file_counts {
  wandb_count: 6
  media_count: 8
}
pusher_stats {
  uploaded_bytes: 44702
  total_bytes: 44702
}

2022-04-09 12:11:53,345 INFO    MainThread:44436 [wandb_run.py:_on_finish():1831] got exit ret: file_counts {
  wandb_count: 6
  media_count: 8
}
pusher_stats {
  uploaded_bytes: 44702
  total_bytes: 44702
}

2022-04-09 12:11:53,447 INFO    MainThread:44436 [wandb_run.py:_on_finish():1831] got exit ret: file_counts {
  wandb_count: 6
  media_count: 8
}
pusher_stats {
  uploaded_bytes: 44702
  total_bytes: 44702
}

2022-04-09 12:11:53,549 INFO    MainThread:44436 [wandb_run.py:_on_finish():1831] got exit ret: file_counts {
  wandb_count: 6
  media_count: 8
}
pusher_stats {
  uploaded_bytes: 44702
  total_bytes: 44702
}

2022-04-09 12:11:54,225 INFO    MainThread:44436 [wandb_run.py:_on_finish():1831] got exit ret: file_counts {
  wandb_count: 6
  media_count: 8
}
pusher_stats {
  uploaded_bytes: 44702
  total_bytes: 44702
}

2022-04-09 12:11:54,656 INFO    MainThread:44436 [wandb_run.py:_on_finish():1831] got exit ret: done: true
exit_result {
}
file_counts {
  wandb_count: 6
  media_count: 8
}
pusher_stats {
  uploaded_bytes: 44702
  total_bytes: 44702
}
local_info {
}

2022-04-09 12:11:55,858 INFO    MainThread:44436 [wandb_run.py:_footer_history_summary_info():2865] rendering history
2022-04-09 12:11:55,858 INFO    MainThread:44436 [wandb_run.py:_footer_history_summary_info():2894] rendering summary
2022-04-09 12:11:55,862 INFO    MainThread:44436 [wandb_run.py:_footer_sync_info():2822] logging synced files
2022-04-09 12:11:55,868 INFO    MainThread:44436 [wandb_init.py:init():492] starting backend
2022-04-09 12:11:55,868 INFO    MainThread:44436 [backend.py:_multiprocessing_setup():99] multiprocessing start_methods=fork,spawn,forkserver, using: spawn
2022-04-09 12:11:55,875 INFO    MainThread:44436 [backend.py:ensure_launched():219] starting backend process...
2022-04-09 12:11:55,921 INFO    MainThread:44436 [backend.py:ensure_launched():224] started backend process with pid: 60005
2022-04-09 12:11:55,923 INFO    MainThread:44436 [wandb_init.py:init():501] backend started and connected
2022-04-09 12:11:55,958 INFO    MainThread:44436 [wandb_run.py:_label_probe_notebook():947] probe notebook
2022-04-09 12:11:55,958 INFO    MainThread:44436 [wandb_run.py:_label_probe_notebook():957] Unable to probe notebook: 'NoneType' object has no attribute 'get'
2022-04-09 12:11:55,958 INFO    MainThread:44436 [wandb_init.py:init():565] updated telemetry
2022-04-09 12:11:56,033 INFO    MainThread:44436 [wandb_init.py:init():596] communicating run to backend with 30 second timeout
2022-04-09 12:11:56,912 INFO    MainThread:44436 [wandb_run.py:_on_init():1759] communicating current version
2022-04-09 12:11:56,973 INFO    MainThread:44436 [wandb_run.py:_on_init():1763] got version response upgrade_message: "wandb version 0.12.14 is available!  To upgrade, please run:\n $ pip install wandb --upgrade"

2022-04-09 12:11:56,973 INFO    MainThread:44436 [wandb_init.py:init():625] starting run threads in backend
2022-04-09 12:11:57,815 INFO    MainThread:44436 [wandb_run.py:_console_start():1733] atexit reg
2022-04-09 12:11:57,816 INFO    MainThread:44436 [wandb_run.py:_redirect():1606] redirect: SettingsConsole.WRAP
2022-04-09 12:11:57,816 INFO    MainThread:44436 [wandb_run.py:_redirect():1643] Wrapping output streams.
2022-04-09 12:11:57,818 INFO    MainThread:44436 [wandb_run.py:_redirect():1667] Redirects installed.
2022-04-09 12:11:57,818 INFO    MainThread:44436 [wandb_init.py:init():664] run started, returning control to user process
